app/livechat/server/config.js

```diff
@@ -80,6 +80,13 @@ Meteor.startup(function() {
 	settings.add('Livechat_allow_switching_departments', true, { type: 'boolean', group: 'Livechat', public: true, i18nLabel: 'Allow_switching_departments' });
 	settings.add('Livechat_show_agent_email', true, { type: 'boolean', group: 'Livechat', public: true, i18nLabel: 'Show_agent_email' });
 
+	settings.add('Livechat_conversation_finished_message_enabled', true, {
```

Comment:
```suggestion
	settings.add('Livechat_request_comment_when_closing_conversation', true, {
```


request comment when ending live chat conversation

Reply:
@knrt10, once we change the name of the setting, you need to change it's translations string as well.

Reply:
ok, let me update